screenpro/ngs/__init__.py

```diff
@@ -195,7 +195,7 @@ def get_counts_matrix(self, fastq_dir, samples, get_recombinant=False, cas_type=
                 if get_recombinant:
                     raise ValueError("Recombinants are not applicable for single guide design!")
                 if protospacer_length == 'auto':
-                    protospacer_length = self.library['protospacer'].str.lengths().unique().to_list()[0]
+                    protospacer_length = self.library['protospacer'].str.len_bytes().unique().to_list()[0]
```

Comment:
@isabelleliuu – is this specific after an specific polars version? It would be helpful if we specify version (e.g. `polars>=*.*`)

Reply:
@abearab based on the polars github releases ([https://github.com/pola-rs/polars/releases/tag/py-0.19.8]) this change was made with the Python Polars 0.19.8 release